### DISCUSSION TITLE
exifから取れない撮り方を書くことについて

### DISCUSSION THREAD

**Discussion by masibw** [jphacks/F_2111#52] [Ideas]
今は title, descriptionを入力してもらってると思います．

`description` というのが写真の説明なのか，写真の撮り方の説明なのかによって変わってきますが，以下のことを書けるようにしたいです

- exifからは取れない撮り方についての説明
  -  朝5時しかこの景色は見れません！
  - １週間通ってやっと撮れました

みたいな

**Comment by takurinton** [comment]
なるほど、良さそう。

**Reply by masibw** [reply]
どう実装します？

あらかじめ 
- 撮りやすさ

みたいな項目をこちらで用意して作るか、任意記入欄を作るか…
